### DISCUSSION TITLE
four-oh-four error

### DISCUSSION THREAD

**Discussion by ledangvu96** [spencerwooo/onedrive-vercel-index#632] [Q&A]
Hi, i got this error when enter my site

{"error":{"error":{"code":"itemNotFound","message":"Item does not exist","innerError":{"date":"2022-05-04T09:54:01","request-id":"b87e26d1-047d-491d-b3e2-fde0f01bd94a","client-request-id":"b87e26d1-047d-491d-b3e2-fde0f01bd94a"}}}}

![image](https://user-images.githubusercontent.com/103246001/166660484-69a9f096-2b0c-475c-b585-5ddee8208b9f.png)

**Comment by turing0** [comment]
I have this error, do u solve this?

**Reply by minterjia** [reply]
解决了吗

**Comment by myl7** [comment]
Any extra Vercel function runtime log, or demo (which is better)? It is hard (nearly impossible) to find out the true reason unless more information is provided.

**Reply by turing0** [reply]
https://onedrive-vercel-index-turing0.vercel.app/
You can see error in this link.

**Reply by myl7** [reply]
It seems even the Vercel functions do not run properly. You may recheck Vercel function log to ensure it.
I would suggest you redeploy the app. There are people who have similar 404 problems but resolved them with redeploying.
